db/migrate/20180703131647_add_interactive_to_image_question.rb

```diff
@@ -0,0 +1,23 @@
+class AddInteractiveToImageQuestion < ActiveRecord::Migration
+  def up
+    add_column :embeddable_image_questions, :interactive_id, :integer
+    add_column :embeddable_image_questions, :interactive_type, :string
+
+    Embeddable::ImageQuestion.find_each(batch_size: 10) do |iq|
```

Comment:
To be safe this file should define `Embeddable::ImageQuestion` and all of the other models it is using. That way if we remove this class in the future the migration can still run.   An example is here: https://github.com/concord-consortium/lara/blob/a58e866b9a954b72002fd0d44b34c1f5447bcafe/db/migrate/20180619214822_fixup_page_item_positions.rb#L1-L9 Unfortunately in this case that requires several models. But I think it is worth the effort. It also makes it more it explicit which relationships of the models are being used by the migration.